onnxscript/function_libs/torch_lib/ops/common.py

```diff
@@ -0,0 +1,37 @@
+"""Common operators shared in the torchlib library."""
+
+import onnxscript
+import onnxscript.values
+from onnxscript import BOOL, INT64
+from onnxscript import opset18 as op
+from onnxscript.function_libs.torch_lib import _constants, tensor_typing
+
+DOMAIN = f"{_constants.DOMAIN}.common"
```

Comment:
Are we going to keep the domain name separate if #834 is fixed?

Reply:
I expect that because the common functions are not torch functions.